pal::hermit::time::SystemTime is not compiling in libstd for Hermit ABI

<!--
Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
along with any information you feel relevant to replicating the bug.
-->

I tried running these commands:

```
$ cargo new --bin herm-test
$ cd herm-test
$ cargo +nightly check -Z build-std --target x86_64-unknown-hermit
```

However, the build failed with the following error:

```
error[E0560]: struct `pal::hermit::time::SystemTime` has no field named `t`
   --> /home/jtnunley/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys/pal/hermit/time.rs:216:46
    |
211 | pub struct SystemTime(Timespec);
    |            ---------- `pal::hermit::time::SystemTime` defined here
...
216 |     pub const MAX: SystemTime = SystemTime { t: Timespec::MAX };
    |                                              ^ field does not exist
    |
help: `pal::hermit::time::SystemTime` is a tuple struct, use the appropriate syntax
    |
216 -     pub const MAX: SystemTime = SystemTime { t: Timespec::MAX };
216 +     pub const MAX: SystemTime = pal::hermit::time::SystemTime(/* Timespec */);
    |

error[E0560]: struct `pal::hermit::time::SystemTime` has no field named `t`
   --> /home/jtnunley/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys/pal/hermit/time.rs:218:46
    |
211 | pub struct SystemTime(Timespec);
    |            ---------- `pal::hermit::time::SystemTime` defined here
...
218 |     pub const MIN: SystemTime = SystemTime { t: Timespec::MIN };
    |                                              ^ field does not exist
    |
help: `pal::hermit::time::SystemTime` is a tuple struct, use the appropriate syntax
    |
218 -     pub const MIN: SystemTime = SystemTime { t: Timespec::MIN };
218 +     pub const MIN: SystemTime = pal::hermit::time::SystemTime(/* Timespec */);
    |
```

### Meta
<!--
If you're using the stable version of the compiler, you should also check if the
bug also exists in the beta or nightly versions.
-->

`rustc --version --verbose`:
```
rustc 1.94.0-nightly (806c2a35d 2025-12-19)
binary: rustc
commit-hash: 806c2a35dcc1db7867864fea0a7a65554a5238b7
commit-date: 2025-12-19
host: x86_64-unknown-linux-gnu
release: 1.94.0-nightly
LLVM version: 21.1.8
```

--- a/library/std/src/sys/pal/hermit/time.rs
+++ b/library/std/src/sys/pal/hermit/time.rs
@@ -213,9 +213,9 @@ pub struct SystemTime(Timespec);
 pub const UNIX_EPOCH: SystemTime = SystemTime(Timespec::zero());
 
 impl SystemTime {
-    pub const MAX: SystemTime = SystemTime { t: Timespec::MAX };
+    pub const MAX: SystemTime = SystemTime(Timespec::MAX);
 
-    pub const MIN: SystemTime = SystemTime { t: Timespec::MIN };
+    pub const MIN: SystemTime = SystemTime(Timespec::MIN);
 
     pub fn new(tv_sec: i64, tv_nsec: i32) -> SystemTime {
         SystemTime(Timespec::new(tv_sec, tv_nsec))

